tap_google_analytics/client.py

```diff
@@ -13,12 +13,36 @@
 
 LOGGER = singer.get_logger()
 
+def is_retryable_403(e):
+    response = e.response
+    if not _is_json(response):
+        return False
+
+    retryable_errors = {"userRateLimitExceeded", "rateLimitExceeded", "quotaExceeded"}
+    error_reasons = {error.get('reason') for error in response.json().get('errors',[])}
+
+    if any(error_reasons.intersection(retryable_errors)):
```

Comment:
Do you need to do any here? I think if you remove it it's still truthy.

Also, instead of 

```
if condition:
   return True

return False
```

Can you instead just
```
return bool(condition)
```
?

Reply:
Bool is only if you really want it to be cast to a boolean. If not just return the condition.

Reply:
That's all true, but I'm leaving it verbose and using `any()` here for readability. Mostly just because this function being negative is confusing enough without having to discern the boolean last-line of the function.

Reply:
Sounds good